third_party/docfx-doclet-143274/src/main/java/com/microsoft/util/Utils.java

```diff
@@ -250,4 +252,19 @@ public List<Element> getImplementedMethods(
   public List<TypeElement> getImplementedInterfaces(TypeElement element) {
     return element.getInterfaces().stream().map(e -> asTypeElement(e)).collect(Collectors.toList());
   }
+
+  // Use to get the recommended package URL for Package Overview
+  public static String[] extractPackageBaseURIBeforeVersion(String input) {

```

Comment:
Prefer Collections over Arrays. (see go/ej3e-28 and go/java-practices/choosing-collections for details)

Reply:
Thank you for the reference! Refactored to be an ImmutableList